CMakeLists.txt

```diff
@@ -138,10 +141,11 @@ ENDIF(NOT DISABLE_TENSORFLOW)
 LIST(LENGTH FILTER_DEP FILTER_DEP_LEN)
 
 IF (SINGLE_BINARY)
-	ADD_LIBRARY(nnstreamerStatic STATIC gst/nnstreamer/nnstreamer.c ${OBJECTS})
+	ADD_LIBRARY(nnstreamerStatic STATIC gst/nnstreamer/nnstreamer.c ${OBJECTS} gst/nnstreamer/tensor_common.c gst/nnstreamer/tensor_meta.c)
```

Comment:
It's strange. Do you want to generate a different file name for a static library as follows? 
- libnnstreamerStatic.a
- nnstreamer.so

The same file name style is better than a different file name structure between static and shared libary.

Reply:
I made a mistake. It's okay.